Dockerfile.cross

```
@@ -14,9 +14,10 @@ ENV PKG_CACHE_PATH=/fetched
 
 RUN git clone -b master --single-branch https://github.com/vercel/pkg-fetch.git /build
 WORKDIR /build
+RUN yarn add babel-cli
 RUN yarn && yarn run babel
```

```suggestion
RUN npm install
RUN npm run build
```

Could you try with this?